tensorflow_cloud/containerize.py

```diff
@@ -227,10 +227,14 @@ def _get_logs(logs_generator, name):
     for line in logs_generator:
         try:
             unicode_line = line.decode('utf-8').strip()
+            logger.info(unicode_line)
         except UnicodeError:
-            logger.warn('Unable to decode logs.')
-        line = json.loads(unicode_line)
-        if line.get('error'):
-            raise RuntimeError(
-                'Docker image {} failed: {}'.format(
-                    name, str(line.get('error'))))
+            logger.warning('Unable to decode logs.')
+        try:            
+            line = json.loads(unicode_line)
+            if line.get('error'):
+                raise RuntimeError(
+                    'Docker image {} failed: {}'.format(name, str(
+                        line.get('error'))))
+        except json.decoder.JSONDecodeError:
+            pass
```

Comment:
can we raise an error here?

Reply:
ack. Raised a RuntimeError. Thanks for the feedback